.github/DELETE_ME_BEFORE_MERGE_TO_MAIN/DELETE_ME.py

```diff
@@ -0,0 +1,6 @@
+print("Test")
+data = {
+    'a': 123,
+    'b': 456,
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad': 123 + 4567 + 976843576985432798652492552 + 98734589762345978
+}
```

Comment:
```suggestion
```
Take a look at the filename for explanation here.

Reply:
Removed in squash